components/display_drivers/include/display_drivers.hpp

```diff
@@ -96,10 +96,17 @@ static void init_pins(gpio_num_t reset, gpio_num_t data_command, gpio_num_t back
 
   using namespace std::chrono_literals;
   // Reset the display
+#if CONFIG_BSP_ESP32_S3_BOX_3
+  gpio_set_level(reset, 1);
+  std::this_thread::sleep_for(100ms);
+  gpio_set_level(reset, 0);
+  std::this_thread::sleep_for(100ms);
+#else
```

Comment:
Given that this isn't a config that this component provides, perhaps we can update the API for the display drivers component to enable you to configure the polarity of the reset pin, since that seems like it would provide the functionality you need. Would that work for you?

Reply:
I am currently adapting the BOX-3 and making secondary developments for your open-source project esp-box-emu. There have been some minor changes to the pins of BOX-3, including the need to invert the polarity of the lcd reset pin. I think providing an API for the polarity of the reset pin is feasible, but there will also be some challenges. By the way, your open-source project esp-box-emu is a great project.

Reply:
Sounds good - please see #107 and see if that helps :)

At some point I'll probably rename some of those variables to make them clearer, but for now I've added a `reset_value` member to the `display_drivers::Config` structure that should allow you to do what you need to for the BOX-3

Reply:
Sure, the reset_value can help me adapt to BOX-3. Thank you very much for the work you've done, and thank you for your open-source project.